src/.storybook/preview.js

```diff
@@ -19,7 +19,12 @@ addParameters({
     // Disable Docs globally for now until we got time to update them.
     // We are still able to add Docs in component level.
     // https://github.com/storybookjs/storybook/blob/next/addons/docs/docs/docspage.md#replacing-docspage
-    page: null
+    page: null,
+    // We'd better to use inline stories to replace iframe.
+    // However multi stories with same knob name in one docs page will be overridden.
+    // We need to migrate knobs to control in v6.
+    // https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs
+    // inlineStories: true
```

Comment:
After we upgraded storybook from v5 to v6, we can use inlineStories with `Control`. We need to migrate all knobs.